pkg/minishift/openshift/oc.go

```diff
@@ -68,3 +90,114 @@ func AddContextForProfile(profile string, ip string, username string, namespace
 	}
 	return nil
 }
+
+// Get the route for service
+func GetServiceURL(service, namespace string, https bool) (string, error) {
+	urlScheme := "http://"
+	if https {
+		urlScheme = "https://"
+	}
+
+	if namespace == "default" {
```

Comment:
@hferentschik , Seems to be getting error saying "No resource found" when querying.